js/main.js

```diff
@@ -1,2 +1,28 @@
 /* global data */
 /* exported data */
+var photoUrl = document.querySelector('.photo-Url');
+var img = document.querySelector('.img');
+photoUrl.addEventListener('input', updateUrl);
+function updateUrl(event) {
+  img.setAttribute('src', photoUrl.value);
+}
+
+var form = document.querySelector('.form');
+var dataEntries = data.entries;
+form.addEventListener('submit', submit);
+
+function submit(event) {
+  event.preventDefault();
+  var title = form.elements.title.value;
+  var url = form.elements.url.value;
+  var notes = form.elements.notes.value;
+  var formObject = {
+    title: title,
+    url: url,
+    notes: notes
+  };
+  dataEntries.push(formObject);
```

Comment:
We want to record the entries in reverse chronological order, so use unshift() instead of push() here.